README.md

```diff
@@ -1,6 +1,6 @@
 # ethereum-validators-monitoring (aka balval)
 
-Consensus layer validators monitoring bot, that fetches Lido Node Operator keys
+Consensus layer validators monitoring bot, that fetches Lido or User Node Operator keys
```

Comment:
```suggestion
Consensus layer validators monitoring bot, that fetches Lido or Custom Users Node Operators keys
```

Reply:
Also add a short note on how to implement custom registry module

Reply:
> Also add a short note on how to implement custom registry module

https://github.com/lidofinance/ethereum-validators-monitoring/blob/feat/custom-validators/README.md#use-custom-validators-list